rskj-core/src/main/java/co/rsk/peg/storage/BridgeStorageAccessorImpl.java

```diff
@@ -18,29 +16,21 @@ public BridgeStorageAccessorImpl(Repository repository) {
 
     @Override
     public <T> T safeGetFromRepository(DataWord keyAddress, RepositoryDeserializer<T> deserializer) {
```

Comment:
Since we are no longer handling potential exceptions, I see no sense in continuing to use the prefix "safe" in all these refactored methods. Wdyt?

Reply:
very good point actually

Reply:
Yeap, just call getFromRepository directly

Reply:
done